src/scanners/css.js

```diff
@@ -26,7 +26,15 @@ export default class CSSScanner extends BaseScanner {
                 }
 
                 this.validatorMessages = this.validatorMessages.concat(
-                  rules[cssRule](rule));
+                  rules[cssRule](
```

Comment:
Would be nice to try and indent this so it's more obviously a function - maybe see about moving the first two args up onto the first line?